src/functionaltests/resources/eiffel_events_for_test.json

```diff
@@ -52,11 +52,8 @@
         "classifier": ""
       }
       ],
-      "gav": {
-        "version": "1.58.0",
-        "artifactId": "sub-system",
-        "groupId": "com.mycompany.myproduct"
-      }
+            
```

Comment:
Shouldn't meta.version be updated from 1.0.0 to 3.0.0? This goes for all Eiffel events we send in our tests.

Reply:
done

Reply:
This file still needs to be updated in the event version fields.